apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/components/template-selector-item.js

```diff
@@ -1,31 +1,39 @@
 /**
  * External dependencies
  */
-
-/**
- * Internal dependencies
- */
+/* eslint-disable import/no-extraneous-dependencies */
+import { isNil, isEmpty } from 'lodash';
+/* eslint-enable import/no-extraneous-dependencies */
 
 /**
  * WordPress dependencies
  */
 import BlockPreview from './block-template-preview';
+/* eslint-disable import/no-extraneous-dependencies */
 import { Disabled } from '@wordpress/components';
+/* eslint-enable import/no-extraneous-dependencies */
 
 const TemplateSelectorItem = props => {
 	const {
 		id,
 		value,
-		help,
 		onFocus,
 		onSelect,
 		label,
 		useDynamicPreview = false,
 		staticPreviewImg,
 		staticPreviewImgAlt = '',
-		blocks = [],
```

Comment:
just curious why we are removing the default value here.

Reply:
Looks like the default is now set here https://github.com/Automattic/wp-calypso/pull/35379/files#diff-3b28d6a34adde060392c645868437593R16

Reply:
Good spot. I think this got introduced when I was adding tests for the presence of `blocks`. It doens't hurt to have the default here afaik. Fixed.